src/tonation.py

```diff
@@ -0,0 +1,19 @@
+import chord
+import copy
+
+
+class Tonation(chord.Chord):
+    def __init__(self, note, timestamp, duration, type):
+        super().__init__(note, timestamp, duration, type)
+        self.tonic = chord.Chord(note, timestamp, duration, type)
+        self.dominant = copy.deepcopy(self.tonic)
+        self.dominant.note += 7
+        self.dominant.type = "dur7"
```

Comment:
This should not be named Tonation. There is only one note in the constructor, and the tonic, dominant and type properties are purely for the guitar chords. Maybe "TriadsFromTonation", or sth like that, since it has nothing to do with the tonation recognition part